userguide/sharing.rst

```diff
@@ -2620,3 +2476,142 @@ Return to
 |ui-options| on the desired file extent, then click :guilabel:`Edit`.
 Set the size to *0* as this causes the iSCSI target to use the new
 size of the file.
+
+
+.. index:: Time Machine
+.. _Creating Authenticated and Time Machine Shares:
+
+Creating Authenticated and Time Machine Shares
+----------------------------------------------
+
+macOS includes the
+`Time Machine <https://support.apple.com/en-us/HT201250>`__ feature
+which can be used to perform automatic back ups. %brand% supports
+Time Machine back ups for both :ref:`AFP <Apple (AFP) Shares>` and
+:ref:`SMB <Windows (SMB) Shares>` shares. The process for creating an
+authenticated share for a user is the same as creating a Time Machine
+share for that user.
+
+Before creating a Time Machine or authenticated share, go to
```

Comment:
This is kind of oddly worded.  It should not say "Before", but should just explain that these types of shares should be created on an :ref:`new dataset<Adding Datasets>`.